core/src/main/scala/org/http4s/instances/KleisliInstances.scala

```diff
@@ -0,0 +1,16 @@
+package org.http4s
+package instances
+
+import cats._
+import cats.data._
+import cats.implicits._
+
+trait KleisliInstances {
+
+  implicit def http4sKleisliSemigroupK[F[_]: SemigroupK, A]: SemigroupK[Kleisli[F, A, ?]] =
```

Comment:
And this is what we can eliminate with typelevel/cats#1928.

Reply:
👍

Reply:
This is in that same awkward position as `KleisliOps`, but it replicates the syntax we want people to have once that cats issue moves forward.  I'm sad but totally okay with this one.